lib/apiScenario/gen/staticApiScenarioGenerator.ts

```diff
@@ -52,13 +38,13 @@ export class StaticApiScenarioGenerator {
       eraseDescription: false,
       filerTopLevelResourceType: true,
       noExternalDependencyResourceType: true,
-      rules: ["resource-put-delete"],
```

Comment:
Is this rule used currently? Is it safe to delete?

Reply:
Currently, we don't use this. Unified pipeline use fixed version of oav. It's safe to delete it.